### ISSUE TITLE
WFINFO_FILTERED_ITEMS and WFINFO_PRICES

### ISSUE THREAD

**Issue by cmaranes** [WFCD/warframe-status#1435]
Hi!

I've noticed that `WFINFO_FILTERED_ITEMS` and `WFINFO_PRICES` are set as environment variables in the code, likely pointing to external API URLs. I'm facing challenges running the code locally due to these unset environment variables.

Would it be possible to provide the URLs or guidance on how to set these environment variables to point to the appropriate APIs? Alternatively, if there's any API documentation or original code related to these variables that I could run locally, that would be helpful as well.

Thank you for your assistance!

**Comment by TobiTenno** [comment]
what issues are you having running the project without them? they should be ignored in the hydration if you don't have them populated.

**Comment by cmaranes** [comment]
Currently, I can run the project without issues. However, when attempting to query the /wfinfo/prices endpoint, I receive a `{ code: 503, error: 'WFInfo Data Services Unavailable' }` response. It seems that the `WFINFO_PRICES` environment variable is not defined, which might be causing this error.

I would like to set up the WFINFO_PRICES environment variable correctly as I'm interested in obtaining the prices for Mods as well. Thanks in advance!

**Comment by TobiTenno** [comment]
this isn't an error, if you read [where that return code is issued from](https://github.com/WFCD/warframe-status/blob/30a5eae0b60a27180ca956c4fa56f44568dbf2c7/src/controllers/wfinfo.js#L27), you'll see this is to protect against actual errors. 

I have no plans to publish the values I use for those variables, and are intentionally not hard-coded or shared. 

Those 2 endpoints have specific usages and provide nothing unique from the rest of the server, you can find _generally_ the same data elsewhere, those 2 are exposed for, as named, WFInfo, and you can query them from the live server if you need to get an example of what their structure is

**Comment by cmaranes** [comment]
Thank you for your prompt response. I'm also interested in retrieving market prices for Mods in the same manner as the equipment prices. While I attempted to use the live server URL as suggested, I noticed that it already returns the filtered items along with their respective prices, excluding the mods ([source](https://github.com/WFCD/warframe-status/blob/30a5eae0b60a27180ca956c4fa56f44568dbf2c7/src/spec/wfinfo.spec.js#L46C1-L47C1)).

Would it be necessary for me to create a custom API to scrape the mods' prices, or is there a more straightforward method to obtain this information? I apologize for my limited understanding on this matter.

Thank you for your guidance!
